### PR TITLE
set checkbox for repeat videos

### PR DESCRIPTION
**Pull request by husnain446**

### CODE REVIEW

File: app/src/main/java/byteshaft/com/recorder/Helpers.java [hunk 1]
```diff
@@ -146,4 +148,14 @@ void setVolume(int level) {
         AudioManager am = (AudioManager) getSystemService(AUDIO_SERVICE);
         am.setStreamVolume(AudioManager.STREAM_MUSIC, level, 0);
     }
+
+    boolean isCheckBoxEnabled() {
```
Comment (om26er) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: What checkbox is enabled ? You need to change this method to be called something like: isRepeatEnabled()

File: app/src/main/java/byteshaft/com/recorder/Helpers.java [hunk 2]
```diff
@@ -146,4 +148,14 @@ void setVolume(int level) {
...
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
```
Comment (om26er) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: Can you move the PreferenceManager call to a method, along the lines getPreferenceManager()

File: app/src/main/java/byteshaft/com/recorder/Helpers.java [hunk 3]
```diff
@@ -146,4 +148,14 @@ void setVolume(int level) {
...
+        return sharedPreferences.getBoolean("CheckBox", false);
```
Comment (om26er) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: The setting key "CheckBox" should be changed to something like: repeatEnabled

File: app/src/main/java/byteshaft/com/recorder/Helpers.java [hunk 4]
```diff
@@ -146,4 +148,14 @@ void setVolume(int level) {
...
+    }
+
+    void settingForCheckBox(boolean enable) {
```
Comment (om26er) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: What checkBox ? Method names should be as descriptive as possible. You could look into a few other java projects on how method names can be made description.

You could change this method to something like: setRepeatEnabled()

File: app/src/main/java/byteshaft/com/recorder/Helpers.java [hunk 5]
```diff
@@ -146,4 +148,14 @@ void setVolume(int level) {
...
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
```
Comment (om26er) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: Same as suggested above, write a new method to get preference manager.

File: app/src/main/java/byteshaft/com/recorder/SettingFragment.java [hunk 6]
```diff
@@ -5,12 +5,31 @@
...
+public class SettingFragment extends Fragment implements CheckBox.OnCheckedChangeListener {
+    
+    private Helpers mHelpers;
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment, container, false);
+        View view = inflater.inflate(R.layout.fragment, container, false);
+        CheckBox checkbox = (CheckBox) view.findViewById(R.id.ch);
```
Comment (om26er) [inline app/src/main/java/byteshaft/com/recorder/SettingFragment.java]: Again, what is a checkbox for a variable name ? It does not mean anything. Please change the variable name to be descriptive.

File: app/src/main/res/layout/fragment.xml [hunk 7]
```diff
@@ -5,10 +5,10 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <TextView
-        android:id="@+id/secondFragment"
-        android:text="second"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content" />
+    <CheckBox
+        android:id="@+id/ch"
```
Comment (om26er) [inline app/src/main/res/layout/fragment.xml]: 'ch' whaaaat ? Change that to be descriptive.

File: app/src/main/java/byteshaft/com/recorder/Helpers.java (hunk 1 above)
Comment (husnain446) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: fix

File: app/src/main/java/byteshaft/com/recorder/Helpers.java (hunk 3 above)
Comment (husnain446) [inline app/src/main/java/byteshaft/com/recorder/Helpers.java]: fixed

File: app/src/main/java/byteshaft/com/recorder/SettingFragment.java (hunk 6 above)
Comment (husnain446) [inline app/src/main/java/byteshaft/com/recorder/SettingFragment.java]: fixed

**Pull request merged**